scripts/generate-docs/MarkdownBuilder.ts

```diff
@@ -39,6 +39,15 @@ export default class MarkdownBuilder {
     this.append("");
     return this;
   }
+  public selectList(options: Map<string, string>, id: string) {
+    this.append(`<select id="${id}">`);
+    this.append(`<option default value=""></option>`);
+    options.forEach((option, value) => {
+      this.append(`<option value="${value}">${option}</option>`)
```

Comment:
It would be great to have a visual indicator for which languages are supporting and do have examples versus those supported without examples.

Currently:

![image](https://user-images.githubusercontent.com/1885333/38522431-f3864150-3c1e-11e8-8bd2-e519b4666b57.png)